src/components/addressbook/form.js

```diff
@@ -19,7 +19,7 @@ const Render = ({ handleSubmit, blockAddress, invalid, pristine, submitting, can
                         component={renderTextField}
                         type="text"
                         label="Address Name"
-                        validate={ required } />
+                        disabled />
```

Comment:
In this [bug](https://github.com/ethereumproject/emerald-wallet/issues/226) he mentions *"Address name should not be a required field. _After adding an address name you cannot remove it again._ You should not be forced to have an address name."*

So updates are not possible to the address field.